Violin.m

```diff
@@ -197,6 +197,7 @@
             obj.ShowData = args.ShowData;
             obj.ShowNotches = args.ShowNotches;
             obj.ShowMean = args.ShowMean;
+            uistack(obj.ViolinPlot,'bottom');
```

Comment:
Why is the violin plot stacked to the bottom? I think I would generally expect the last plot to end up on top.

Reply:
It's mainly to help with selecting the scatter points using the data cursor or the select tool in the figure panel. With the violin plot on top, I can only select the violin plot and it is rather annoying to select the underlying scatter points.

Reply:
Then we should change the plot order and plot the scatter points last. But stacking the Violin to the bottom is counterintuitive and might break plots.

Reply:
Plot order is more of a visual feature, unless I am mistaken and you use the plot order for another function? I do not think it is that counterintuitive to stack Violin to the bottom using `uistack` since that is the purpose of that function.

Reply:
Imagine someone plotting first a few lines, then a Violin. The Violin should definitely be atop the lines. We can not mess with `uistack`.